tlsutil/config.go

```diff
@@ -572,7 +580,14 @@ func (c *Configurator) commonTLSConfig(state listenerConfig, cfg ListenerConfig,
 
 	// Set the cipher suites
 	if len(cfg.CipherSuites) != 0 {
-		tlsConfig.CipherSuites = cfg.CipherSuites
+		// TODO: is it safe to ignore the error case here?
+		// Should be checked on input, same as tlsConfig.MinVersion
+
+		// FIXME: move cipherSuiteLookup to be called externally, maybe
+		// in agent/config/runtime parsing before the tlsutil.Config struct is
+		// created?
```

Comment:
Yep agree, this should be done up-front rather than when we're trying to use the cipher suites.

Reply:
Looking at this again, this actually feels like an appropriate place to handle the conversion from `types.TLSCipherSuite` to `crypto/tls` values for creating the `tls.Config` object.